.github/workflows/documentation-checks.yaml

```diff
@@ -30,6 +30,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Set up Python
```

Comment:
To test this I did the following:

- Reverted my workaround so that this workflow fails again in my project ([PR](https://github.com/canonical/craft-cli/pull/198));
- Pointed my project to this fork/branch;
- Verified that it now passes! yay